onnxscript/converter_test.py

```diff
@@ -615,14 +616,25 @@ def inc_alpha_expanded(A: FLOAT[...], alpha: int) -> FLOAT[...]:
     def test_none_attribute(self):
         @script()
         def explicit_none(X):
-            return op.Squeeze(X, axes=None)
+            return op11.Squeeze(X, axes=None)
```

Comment:
Was there a reason the change the opset?

Reply:
If it’s important, would a comment help to remind future readers?

Reply:
It's not critical. I could have used some dummy op for this test, since it was testing the use of None as an attribute value. "axes" was changed from an attribute to input for Squeeze in a later opset. Previously we were not using the opschema to identify attributes, but now we are, so it was getting mapped to an input instead of attribute, defeating the purpose of the test-case.

Reply:
Done